### PR TITLE
rename and tweak the liberty configDropin for working with keycloak JWTs

### PR DESCRIPTION
**Pull request by lmsurpre**
Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>

### CODE REVIEW

**Comment by lmsurpre** [comment]
@ibmatc sure, I nominate @tbieste ...what do we need to do to get him added?

File: data-access/fhir/configDropins/overrides/keycloakJwt.xml [hunk 1]
```diff
@@ -1,9 +1,7 @@
 <server description="fhir-server">
-
-    <!-- Enable features -->
     <featureManager>
         <!-- mpJwt-1.1 is already enabled in the default server.xml, but it doesn't hurt to repeat it here -->
-        <feature>mpJwt-1.1</feature>
+        <feature>mpJwt-1.2</feature>
```
Comment (tbieste) [inline data-access/fhir/configDropins/overrides/keycloakJwt.xml:4]: The comment above says 1.1, but this is 1.2. Please confirm which version is correct.

Comment (lmsurpre) [reply data-access/fhir/configDropins/overrides/keycloakJwt.xml:4]: good catch.  fixed the comment.

**Review by tbieste — APPROVED** [review]
LGTM

**Pull request merged**